Rocket.Chat/Views/EmojiPicker/EmojiPicker.swift

```diff
@@ -33,6 +39,16 @@ class EmojiPicker: UIView {
         }
     }
 
+    var recentCategory: (name: String, emojis: [Emoji]) {
+        // remove invalid custom emojis
+        let recentEmojis = self.recentEmojis.filter {
+            guard let imageUrl = $0.imageUrl else { return true }
```

Comment:
Wouldn't be `return false` here?

Reply:
If imageUrl is not nil, means it's not a custom emoji and we don't need to check if it's invalid, so we return true for the filter.

Reply:
Gotcha! What do you think about creating a property/method/getter called `isCustomEmoji` and then use it? I think it'll be more clear to understand later 😂